spec/controllers/login_controller_spec.rb

```diff
@@ -110,6 +116,12 @@
       get :forgot_password
       expect(response).to redirect_to('/')
     end
+
+    it "doesn't allow submission from logged in users" do
+      post :login, params: { email: user.email, password: 'asdf' }
+      post :reset_password, params: { email: user.email, password: 'asdf' }
```

Comment:
reset_password doesn't take a password

```suggestion
      post :reset_password, params: { email: user.email }
```